hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/IncrementalBlockReportManager.java

```diff
@@ -304,4 +310,4 @@ void clearIBRs() {
   int getPendingIBRSize() {
     return pendingIBRs.size();
   }
-}
\ No newline at end of file
+}
```

Comment:
can rollback this change

Reply:
Done. Actually took me quite a while to find the setting to disable it in IDE